06-Modelling-Rigid-Bodies/01-Introduction-to-Rigid-Bodies.ipynb

```diff
@@ -0,0 +1,839 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    snapshot.particles.moment_inertia = [I[0, 0], I[1, 1], I[2, 2]] * N_particles
</pre><p>Suggestion:</p><pre class="ql-syntax" spellcheck="false">snapshot.particles.moment_inertia = numpy.tile(numpy.diag(I), (N_particles, 1))
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/61/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
<p>I could. I'll keep the list expansion for consistency and clarity for new users.</p>